src/map/party.cpp

```diff
@@ -1217,7 +1217,7 @@ uint32 CParty::GetTimeLastMemberJoined()
 {
     auto* PLeader = dynamic_cast<CCharEntity*>(CParty::GetLeader());
 
-    return (uint32)std::chrono::time_point_cast<std::chrono::seconds>(PLeader->m_TimeLastMemberJoined).time_since_epoch().count();
+    return (uint32)std::chrono::time_point_cast<std::chrono::seconds>(PLeader->LeaderCreatedPartyTime).time_since_epoch().count();
```

Comment:
m_LeaderCreatedPartyTime?

Reply:
Honestly have no idea what happend here but glad you spotted it.